app/src/main/java/com/sjh/autosummary/core/network/retrofit/RetrofitGptApiHolder.kt

```diff
@@ -0,0 +1,44 @@
+package com.sjh.autosummary.core.network.retrofit
+
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import com.sjh.autosummary.BuildConfig
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import java.util.concurrent.TimeUnit
+
+class RetrofitGptApiHolder(
+    json: Json,
+) {
+    private companion object {
+        const val BASE_URL = "https://api.openai.com/v1/"
+        const val GPT_API_KEY = BuildConfig.GPT_API_KEY
+    }
+
+    val retrofitGptApi: RetrofitGptApi =
+        Retrofit
+            .Builder()
+            .baseUrl(BASE_URL)
+            .client(
+                OkHttpClient
+                    .Builder()
+                    .connectTimeout(30, TimeUnit.SECONDS)
+                    .readTimeout(30, TimeUnit.SECONDS)
+                    .writeTimeout(30, TimeUnit.SECONDS)
+                    .addInterceptor { chain ->
+                        val newRequest = chain
+                            .request()
+                            .newBuilder()
+                            .addHeader("Content-Type", "application/json")
+                            .addHeader("Accept", "application/json")
```

Comment:
```suggestion
                            .addHeader("Content-Type", "application/json; charset=utf-8")
                            .addHeader("Accept", "application/json; charset=utf-8")
```
utf-8을 붙여주시는게 좋습니다. 왜 인지는 스스로 찾아보세요.

Reply:
넵! 조사해보겠습니다!